frontend/src/scenes/plugins/edit/PluginSource.tsx

```diff
@@ -12,40 +12,41 @@ import SCAFFOLD_errors from '!raw-loader!@posthog/plugin-scaffold/dist/errors.d.
 // @ts-ignore
 import SCAFFOLD_types from '!raw-loader!@posthog/plugin-scaffold/dist/types.d.ts'
 
-const defaultSource = `// Learn more about plugins at: https://posthog.com/docs/plugins/overview
-import { Plugin } from '@posthog/plugin-scaffold'
+const defaultSource = `
+// Learn more about plugins at: https://posthog.com/docs/plugins/build/overview
 
-type MyPluginType = Plugin<{
-  config: {
-    username: string
-  },
-  global: {},
-}>
+/* Runs on every event */
+export function processEvent(event, { config }) {
+    // Some events (like $identify) don't have properties
+    if (event.properties) {
+        event.properties['hello'] = \`Hello \${config.name}\`
+    }
+    // Return the event to ingest, return nothing to discard  
+    return event
+}
+
+/* Runs once on plugin installation */
+export function setupPlugin (meta) {
 
-const MyPlugin: MyPluginType = {
-  setupPlugin: async (meta) => {
-    
-  },
-  onEvent: async (event, meta) => {
-    console.log(\`Event \${event.event} has been processed!\`)
-  },
 }
 
-export default MyPlugin
-`
+/* Runs once every full hour */
+// function runEveryHour(meta) {
+//     const weather = await (await fetch('https://weather.example.api/?city=New+York')).json()
+//     posthog.capture('weather', { degrees: weather.deg, fahrenheit: weather.us })
+// }`
```

Comment:
Feels a bit weird to leave commented-out code in. I think we actually can provide a functioning `runEveryHour` by default